frontend/config/config.ts

```diff
@@ -0,0 +1,35 @@
+import dotenv from 'dotenv';
+
+dotenv.config()
+
+
+export const config = {
+    db: {
+        host: process.env.DB_HOST || '0.0.0.0',
```

Comment:
IMO, use one connection URI environment variable will make it simple and flexible